src/api/app/models/comment.rb

```diff
@@ -1,14 +1,15 @@
 require 'set'
 
 class Comment < ApplicationRecord
-  belongs_to :commentable, polymorphic: true # belongs to a Project, Package or BsRequest
+  belongs_to :commentable, polymorphic: true # belongs to a Project, Package, BsRequest or BsRequestActionSubmit
```

Comment:
We can change `BsRequestActionSubmit` to `BsRequestAction`

Reply:
It's correct. I was referring to the association itself. Just checked that the code is specifically checking `BsRequestActionSubmit` :+1: